alivee/utils/requests/zillow/zillow_deep_search_result_request_meta.py

```diff
@@ -21,3 +23,7 @@ def __init__(self,
             'rentzestimate': rentEstimate
         }
         super(ZillowDeepSearchResultsRequestMeta, self).__init__(service_id, self.PATH, params)
+
+    def make_request(self):
+        response = requests.request(self.method, self.url)
+        return response
```

Comment:
Maybe try to find some existing third party python `.xml` parsing package. AFAIK, `beautifulsoup` is a good choice maybe.